tests/test_memory_storage.py

```
@@ -0,0 +1,1123 @@
+        c.smove("not_a_set", "set2", "member")
+
+
+@pytest.mark.parametrize("r", ["memory_storage", "redisclient"])
+def test_storage_smove_wrong_type_destination(r, request):
+    """Test smove when destination is not a set."""
+    c = request.getfixturevalue(r)
+    c.sadd("set1", "member1")
+    c.set("not_a_set", "value")
+
+    # Try to move to a non-set
+    with pytest.raises(Exception):  # ResponseError
```

Ruff (0.14.1)

Do not assert blind exception: `Exception`

(B017)